tests/test_minimal/test_tools/test_backend_and_dataset_configuration/test_helpers/test_get_default_backend_configuration.py

```diff
@@ -262,3 +263,22 @@ def test_complex_zarr(zarr_nwbfile_path):
 
 """
     assert stdout.getvalue() == expected_print
+
+
+def test_000_ImageSeries():
```

Comment:
I would like to see many more tests here; starting with more basic types like `TimeSeries`, `DynamicTable`, etc. before working our way up to the edge case that is an ImageSeries (external vs. internal mode too on that)

Reply:
Keep in mind the intended point of this function; to start with objects that are either uncompressed, unchuncked, or badly chunked; and then we create a new copy of the file that has better compression and chunking on all applicable datasets